docs/content/configuration/broker.md

```diff
@@ -67,7 +67,6 @@ The broker uses processing configs for nested groupBy queries. And, optionally,
 
 |Property|Description|Default|
 |--------|-----------|-------|
-|`druid.query.groupBy.singleThreaded`|Run single threaded group By queries.|false|
```

Comment:
if we want to be backwards compatible we should have this configuration option define the default mode, but have it overridable at query time.